build/android/gyp/javac.py

```diff
@@ -92,21 +92,7 @@ def DoJavac(
       '-classpath', ':'.join(classpath),
       '-d', classes_dir]
 
-  def GetJavaHome():
-    # Android's boot jar doesn't contain all java 8 classes.
-    # See: https://github.com/evant/gradle-retrolambda/issues/23.
-    # Get the path of the jdk folder by searching for the 'jar' executable. We
-    # cannot search for the 'javac' executable because goma provides a custom
-    # version of 'javac'.
-    # Can't use jar path for macOS though.
-    if sys.platform == 'darwin':
-      return subprocess.check_output(['/usr/libexec/java_home', '-v', '1.8']).strip()
-    jar_path = os.path.realpath(distutils.spawn.find_executable('jar'))
-    return os.path.dirname(os.path.dirname(jar_path))
-
   if bootclasspath:
-    rt_jar = os.path.join(GetJavaHome(), 'jre', 'lib', 'rt.jar')
-    bootclasspath.append(rt_jar)
```

Comment:
Does this still compile and run properly with your related engine patch?

Reply:
Yes - once both patches are approved I will add a buildroot roll to the engine patch

Reply:
The answer is yes.  Cool, TIL